vanilla-framework/vanilla-docs-theme
# Update structure

## Done
Update the structure of the theme. Added Jekyll instead of the demo HTML file.

## QA
- Pull this branch
- Run `rm -rf node_modules`
- Run `npm i`
- Run `gulp jekyll`
- Go to the local URL and check jekyll works although there is no content yet.
